docs/www/quick-start-kubernetes.md

```diff
@@ -14,8 +14,14 @@ To get up and running you need to create a cluster and deploy the Redpanda opera
 - For production or benchmarking, set up a [production deployment](/docs/production-deployment).
 - You can also set up a [Kubernetes cluster with external access](/docs/kubernetes-external-connect).
 
-> **_Note_** - Run a container inside the Kubernetes cluster to communicate with the Redpanda cluster.
-> Currently, a load balancer is not automatically created during deployment by default.
+**_Note_** - Run a container inside the Kubernetes cluster to communicate with the Redpanda cluster.
+Currently, a load balancer is not automatically created during deployment by default.
+
+**_Note_** - The examples on this page assume that you have `developerMode: true` set. 
+If you have `developerMode: false` set, we recommend that you run `rpk redpanda tune all` directly on the host for the optimal configuration. 
```

Comment:
> If you have `developerMode: false` set

=>

If you change developerMode to false.. for the optimal configuration prior to creating any redpanda clusters.